scrape_books/pipelines.py

```diff
@@ -0,0 +1,30 @@
+from scrapy import Spider
+from typing import Any, Dict
+
+
+RATING: Dict[str, int] = {
+    "One": 1,
+    "Two": 2,
+    "Three": 3,
+    "Four": 4,
+    "Five": 5
+}
+
+
+class ScrapeBooksPipeline:
+    def process_item(
+            self, item: Dict[str, Any], spider: Spider
+    ) -> Dict[str, Any]:
+        item["price"] = float(item["price"].replace("£", ""))
```

Comment:
Ensure that the `price` field in the item always contains the currency symbol '£'. Otherwise, this line will raise an error if the symbol is missing.